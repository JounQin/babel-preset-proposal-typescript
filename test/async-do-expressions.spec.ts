import _ from './async-do-expressions'
import { execute } from './helpers'

const proposal = 'async-do-expressions'

test(proposal, async () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(await _()).toEqual([2, 6])
})
