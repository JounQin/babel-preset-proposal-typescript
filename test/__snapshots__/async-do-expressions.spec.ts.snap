// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`async-do-expressions 1`] = `"await is only valid in async function"`;
